src/hooks/useNetwork.ts

```diff
@@ -30,5 +29,5 @@ export default function useNetwork({onReconnect = () => {}}: UseNetworkProps = {
         prevOfflineStatusRef.current = isOffline;
     }, [isOffline]);
 
-    return {isOffline};
+    return {isOffline: networkStatus === CONST.NETWORK.NETWORK_STATUS.UNKNOWN ? false : isOffline};
```

Comment:
It'd be good to add a comment to explain why we're doing this. Rather than just returning the value of isOffline directly.